modules/cart/src/Plugin/views/field/EditQuantity.php

```diff
@@ -88,18 +88,27 @@ public function viewsForm(array &$form, FormStateInterface $form_state) {
 
     $form[$this->options['id']]['#tree'] = TRUE;
     foreach ($this->view->result as $row_index => $row) {
+      /** @var \Drupal\commerce_order\Entity\OrderItemInterface $order_item */
       $order_item = $this->getEntity($row);
+
+      /** @var \Drupal\Core\Entity\Display\EntityFormDisplayInterface $form_display */
+      $form_display = commerce_get_entity_display('commerce_order_item', $order_item->bundle(), 'form');
```

Comment:
@bojanz in general, I believe we need to have the cart form mimic the order item form display as much as possible. This kind of hackishly does that. Would rather save for a follow up to limit scope.